src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -72,10 +112,19 @@ object TSVFormatter {
         }
         columnPosition -> cellValue
     }
-    // The rowMap manipulation sorts the position-value map by the key, converts it to a seq of
-    // tuples, pulls out the second element of the tuple (column value) resulting in a sorted seq
-    // of the original map values.
-    entity.name +: rowMap.toSeq.sortBy(_._1).map(_._2).toIndexedSeq
+    // If there are entities that don't have a value for which there is a known header, that will
+    // be missing in the row. Fill up those positions with empty strings in that case.
+    val completedRowMap: IndexedSeq[(Int, String)] =
```

Comment:
There is an example in the test case. Are you saying you want to see that in a code comment?